README.md

```
@@ -1,5 +1,9 @@
 # MCP Server Demo
 
+[![Set it Free Loop](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2FHemSoft%2Fmcp-server-demo%2Fmain%2Fsfl.json&query=%24.version&prefix=v&label=Set%20it%20Free%20Loop&color=FFD700&style=flat&logo=githubactions&logoColor=white)](https://github.com/HemSoft/set-it-free-loop)
+<!-- SFL_BADGE: auto-updated by deploy-workflow.ps1 -->
+# MCP Server Demo
```

markdownlint-cli2 (0.23.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)